CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - UPD+Chromograph to identify and visualize UPD sites and regions in the chromosomes [#364](https://github.com/nf-core/raredisease/pull/364) and [#366](https://github.com/nf-core/raredisease/pull/366)
 - Added check for presence of case id for each sample in samplesheet [#357](https://github.com/nf-core/raredisease/pull/357)
 
+### Changed
+
+- [#385](https://github.com/nf-core/sarek/pull/385) - Updating tiddit to v3.6.1
```

Comment:
Could you make this entry under version 1.2.0?

Reply:
I'll fix that. When developing on Sarek, we always do a "back to dev" PR as the first thing after each release. The same approach might also be suitable for raredisease.

Reply:
Ah, I see that you do have something like a "back-to-dev" PR.

https://github.com/nf-core/raredisease/pull/384 

It was just a bit late in getting merged. Anyways, no hopefully all good.

Reply:
Yup.. 😄